darwin/utils.py

```diff
@@ -3,7 +3,7 @@
 
 from darwin.config import Config
 
-SUPPORTED_IMAGE_EXTENSIONS = [".png", ".jpeg", ".jpg", ".PNG", ".JPEG", ".JPG"]
+SUPPORTED_IMAGE_EXTENSIONS = [".png", ".jpeg", ".jpg", ".jfif", ".PNG", ".JPEG", ".JPG", ".JFIF"]
```

Comment:
Some Android files use this format: https://en.wikipedia.org/wiki/JPEG_File_Interchange_Format

Reply:
```
def allowed_extension(ext):
    return ext.lower() in SUPPORTED_IMAGE_EXTENSIONS
```